components/breadcrumb/index.css

```diff
@@ -10,123 +10,290 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 
-.spectrum-Breadcrumbs-item--sizeS {
-  @remapvars {
-    find: /--spectrum-breadcrumb(.*)-s-/;
-    filter: color;
-    replace: --spectrum-breadcrumb$1-;
+  [dir="rtl"] & {
+    transform: scaleX(-1);
   }
-}
-.spectrum-Breadcrumbs-item--sizeM {
-  @remapvars {
-    find: /--spectrum-breadcrumb(.*)-m-/;
-    filter: color;
-    replace: --spectrum-breadcrumb$1-;
+
+  .spectrum-Breadcrumbs--compact & {
+    margin-block: var(--mod-breadcrumbs-icon-spacing-block-compact, var(--spectrum-breadcrumbs-icon-spacing-block-compact));
   }
-}
-.spectrum-Breadcrumbs-item--sizeL {
-  @remapvars {
-    find: /--spectrum-breadcrumb(.*)-l-/;
-    filter: color;
-    replace: --spectrum-breadcrumb$1-;
+
+  .spectrum-Breadcrumbs--multiline & {
+    margin-block-start: var(--mod-breadcrumbs-icon-spacing-block-start-multiline, var(--spectrum-breadcrumbs-icon-spacing-block-start-multiline));
+    margin-block-end: var(--mod-breadcrumbs-icon-spacing-block-between-multiline, var(--spectrum-breadcrumbs-icon-spacing-block-between-multiline));
   }
 }
 
 .spectrum-Breadcrumbs-item {
+  box-sizing: border-box;
+  position: relative;
   display: inline-flex;
-  align-items: center;
-  justify-content: flex-start;
+  white-space: nowrap;
 
-  box-sizing: border-box;
-  block-size: var(--spectrum-breadcrumb-list-height);
+  font-family: var(--mod-breadcrumbs-font-family, var(--spectrum-breadcrumbs-font-family));
+  font-size: var(--mod-breadcrumbs-font-size, var(--spectrum-breadcrumbs-font-size));
+  font-weight: var(--mod-breadcrumbs-font-weight, var(--spectrum-breadcrumbs-font-weight));
+  line-height: var(--mod-breadcrumbs-line-height, var(--spectrum-breadcrumbs-line-height));
 
-  position: relative;
+  .spectrum-Breadcrumbs--compact & {
+    font-family: var(--mod-breadcrumbs-font-family-compact, var(--spectrum-breadcrumbs-font-family-compact));
+    font-size: var(--mod-breadcrumbs-font-size-compact, var(--spectrum-breadcrumbs-font-size-compact));
+    font-weight: var(--mod-breadcrumbs-font-weight-compact, var(--spectrum-breadcrumbs-font-weight-compact));
 
-  white-space: nowrap;
-  font-size: var(--spectrum-breadcrumb-item-text-size);
-  font-weight: var(--spectrum-breadcrumb-item-text-font-weight);
-  line-height: var(--spectrum-breadcrumb-item-text-line-height);
+    > .spectrum-ActionButton {
+      margin-block: var(--mod-breadcrumbs-action-button-spacing-block-compact, var(--spectrum-breadcrumbs-action-button-spacing-block-compact));
+    }
+  }
+
+  .spectrum-Breadcrumbs--multiline & {
+    font-family: var(--mod-breadcrumbs-font-family-multiline, var(--spectrum-breadcrumbs-font-family-multiline));
+    font-size: var(--mod-breadcrumbs-font-size-multiline, var(--spectrum-breadcrumbs-font-size-multiline));
+    font-weight: var(--mod-breadcrumbs-font-weight-multiline, var(--spectrum-breadcrumbs-font-weight-multiline));
+
+    &:last-of-type {
+      block-size: auto;
+      inline-size: 100%;
+    }
+
+    > .spectrum-ActionButton {
```

Comment:
Not blocking, would you be able to explain or point to some docs for what `> .` is doing?

Reply:
CSS selector, selects all .spectrum-ActionButton elements that are immediate children (first ancestor, not grandchildren) of the parent. 

So this will select:

.spectrum-Breadcrumbs--compact 
   .spectrum-Breadcrumbs-item 
      .spectrum-ActionButton

But it will not select:

.spectrum-Breadcrumbs--compact 
   .spectrum-Breadcrumbs-item 
      .spectrum-ActionButton
        .spectrum-ActionButton

This selector was already in the CSS, and I try to preserve as much existing CSS as possible as long as it still aligns with the designs and is reflective of up to date code standards.